packages/core/tests/connectionless-proofs.test.ts

```diff
@@ -1,5 +1,6 @@
 import type { SubjectMessage } from '../../../tests/transport/SubjectInboundTransport'
```

Comment:
From what I can see here it looks like there are only connectionless tests for V1, is that intentional?

Reply:
It was a miss and I have added the connectionless tests for V2 now